terraform/variables.tf

```diff
@@ -58,18 +58,9 @@ variable "dataset_id" {
   description = "The BigQuery dataset id to create."
 }
 
-variable "dataset_iam" {
-  description = "IAM member bindings for the BigQuery dataset."
-  type = object({
-    owners  = list(string)
-    editors = list(string)
-    viewers = list(string)
-  })
-  default = {
-    owners  = []
-    editors = []
-    viewers = []
-  }
+variable "dataset_editor_service_account_member" {
```

Comment:
This is kind of specific, what if you need to use this same service account for other permissions other than dataset later?

Reply:
This module requires that these service accounts have these permissions right? Seems like we should be explicit about setting what token minter needs.

Reply:
Eh, I guess. I renamed it to be more generic.